src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1286,26 +1287,34 @@ export class ResultsViewPageStore {
             this.molecularProfilesInStudies
         ],
         invoke:()=>{
-            const MRNA_EXPRESSION = "MRNA_EXPRESSION";
-            const PROTEIN_LEVEL = "PROTEIN_LEVEL";
+            const MRNA_EXPRESSION = AlterationTypeConstants.MRNA_EXPRESSION;
+            const PROTEIN_LEVEL = AlterationTypeConstants.PROTEIN_LEVEL;
+            const METHYLATION = AlterationTypeConstants.METHYLATION;
             const selectedMolecularProfileIds = stringListToSet(this.selectedMolecularProfileIds);
 
             return Promise.resolve(_.sortBy(_.filter(this.molecularProfilesInStudies.result!, profile=>{
-                return (profile.molecularAlterationType === MRNA_EXPRESSION ||
-                        profile.molecularAlterationType === PROTEIN_LEVEL) && profile.showProfileInAnalysisTab;
+                return ((profile.molecularAlterationType === MRNA_EXPRESSION ||
+                        profile.molecularAlterationType === PROTEIN_LEVEL) && profile.showProfileInAnalysisTab) ||
```

Comment:
@angelicaochoa I have this in here for now because otherwise no methylation profiles show up, because none of them are shown in analysis tab. Should I keep it like this, or are you going to change that on the data side?